### PR TITLE
Implemented Min() & Max() functions and updated README.md

### PR DESCRIPTION
**Pull request by SadToBeNep**
Hello, I was using your lib for one of my project, and it was missing the .Min() function, so I added it 

I am just learning C# and Git(Hub) too, so go easy on me if something

PS: Wasn't sure if I should open a Issue before the pull request

### CODE REVIEW

**Comment by dragon7307** [comment]
Hey, Glad you like the project. 
I will have a look at your contribution and then merge it.  
Thank you also for opening a pull request - that makes maintaining an open source project  a whole lot easier!! 

Of course, `Min` cannot be implemented without its counterpart `Max`, so I'll see when I have time to get around to that. 
Since I'm planning to release version **1.3.0** at the end of **february**, this is when your contribution should get shipped into production. 
    
PS: Opening an issue never hurts, but is unnecessary in this case, since your pr gives all needed information.

**Comment by SadToBeNep** [comment]
I also started to work on `Max`, most likely I will have time tomorrow and can push that too up here in another pr

**Comment by Guiorgy** [comment]
@SadToBeNep If you plan to implement `Max` as well, maybe instead of another PR, I'd suggest updating the current title an have this PR for both, since as dragon stated, it doesn't make sence to add one and not the other.

File: src/Extensions/Span/SpanExtensions.Linq.cs [hunk 1]
```diff
@@ -460,6 +460,166 @@ public static BigInteger Average(this Span<BigInteger> source)
...
+
+#if NET5_0_OR_GREATER
 
+        /// <summary> 
+        /// Computes the Average of all the values in <paramref name="source"/>. 
+        /// </summary>  
+        /// <param name="source">The <see cref="Span{Half}"/> to operate on.</param>    
+        /// <returns>The Min out of all the values in <paramref name="source"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null.</exception>
+        public static Half Min(this Span<Half> source)
```
Comment (SadToBeNep) [inline src/Extensions/Span/SpanExtensions.Linq.cs:488]: Some kind of null check is needed

**Comment by dragon7307** [comment]
Great work - I'll merge it into a feature branch. I believe there are some missing overloads and implementing `MinBy` and `MaxBy `probably won't hurt, so I will implement them.

**Comment by dragon7307** [comment]
Also for performance reasons an if construct should be used instead of a ternary expression,to avoid unnecessary reassignments.

**Pull request merged**